### ISSUE TITLE
Vite Plugin

### ISSUE THREAD

**Issue by vadimvera** [ventojs/vento#58]
Vite has gained significant traction as a fast and efficient build tool for modern web development projects. However, Vite currently lacks direct support for the Vento template engine.

I thought it will be great if Vento could release an official Vite plugin to seamlessly integrate the template engine into Vite-powered projects, so developers can harness the power and simplicity of Vento without the hassle of manual setup and configuration.

**Comment by oscarotero** [comment]
Yes, that's a great idea.
I don't have experience developing Vite plugins. Do you want to help with this?

**Comment by oscarotero** [comment]
working on this here: https://github.com/ventojs/rollup-plugin-vento

**Comment by vadimvera** [comment]
@oscarotero, it's great you found time to work on this! :raised_hands:

I will track your progress there and see if I can help at some point.

**Comment by wrapperup** [comment]
Yes actually, since `with` has been removed, this can be integrated with rollup (since it requires strict modules by default). That was one of the original motivations for it. Also, it would be nice to expose `env` in a hook, or the ability to pass in plugins like a normal instance of Vento would.

**Comment by michaelpumo** [comment]
I'd love to see Vento in use for Vite. I have a small static site and it'd be awesome to use Vento with it!
